scarlet/component.py

```diff
@@ -141,6 +119,45 @@ def check_parameters(self):
                 msg = "Component {} Parameter {} is not finite:\n{}".format(self, k, p)
                 raise ArithmeticError(msg)
 
+    def project(self, model=None, frame=None):
+        """Project a model into a frame
+
+
+        Parameters
+        ----------
+        model: array
```

Comment:
What's the use case for specified model?

Reply:
I'm not a fan of the term "project" here either. If this is always the component's model, I'd prefer e.g. `Component.model_for(frame)`, but I think I miss a use case here

Reply:
Sure, I can change the method name. How about `model_to_frame` to make it clear what the method does. It has to take model as an argument because of autograd. BTW: jax would be the same.